api/capi/include/nntrainer.h

```diff
@@ -13,12 +13,16 @@
  */
 /**
  * @file nntrainer.h
- * @date 02 April 2020
+ * @date 08 July 2020
  * @brief NNTrainer C-API Header.
- *        This allows to construct and control NNTrainer Model.
+ *        This allows to construct, control and train NNTrainer Model.
```

Comment:
```
This allows to construct, control and train NNTrainer Model.
```
===>
```
This allows to construct, control, and train a neural network model in Tizen devices.
```

Let's not restrict model format here.

Reply:
done.